examples/bt_move_to.cpp

```diff
@@ -0,0 +1,141 @@
+#include <ros/ros.h>
+
+#include <behaviortree_mtc/initialize_mtc_task.h>
+#include <behaviortree_mtc/create_mtc_current_state.h>
+#include <behaviortree_mtc/create_mtc_move_to.h>
+#include <behaviortree_mtc/create_mtc_move_relative.h>
```

Comment:
Please remove all includes that are not used in this test. E.g. the move relative.